heron/tools/tracker/src/python/handlers/stmgrsregsummaryhandler.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2017 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" stmgrsregsummaryhandler.py """
+import traceback
+import tornado.gen
+import tornado.web
+
+from heron.common.src.python.utils.log import Log
+from heron.proto import tmaster_pb2
+from heron.tools.tracker.src.python.handlers import BaseHandler
+
+# pylint: disable=attribute-defined-outside-init
+class StmgrsRegSummaryHandler(BaseHandler):
+  """
+  URL - /topologies/stmgrsregsummary?cluster=<cluster>&topology=<topology> \
```

Comment:
Why do we still need this endpoint? I thought the plan was to not add it, but instead use /runtimestate to serve this info. Can we remove this?